README.asciidoc

```diff
@@ -357,7 +357,7 @@ Maven.resolver().loadPomFromClassLoaderResource("/path/to/pom.xml").resolve("G:A
 ----
 
 Resolving artifacts defined in effective POM::
-ShrinkWrap Resolvers allows you to artifacts defined with specific scope into list of artifacts to be resolved. This way, you don't need to alter your tests if you change dependencies of your application. You can either use +importDependencies(ScopeType...)+ or convenience methods, that cover the most frequent usages (+importRuntimeDependencies()+, +importTestDependencies()+ and +importRuntimeAndTestDependencies()+:
```

Comment:
Yeah, the beginning of the sentence is weird. I would write something like:

> ShrinkWrap Resolvers allows you to import artifacts from your POM file, select them by specific scopes and resolve them. This way, you don't need to resolve every single dependency separately or alter your tests if you change dependencies of your application. You can either use +importDependencies(ScopeType...)+ or convenience methods, that cover the most frequent usages (+importCompileAndRuntimeDependencies()+, +importRuntimeDependencies()+, +importTestDependencies()+ and +importRuntimeAndTestDependencies()+):

Does it make more sense?